pkg/apis/core/validation/shoot.go

```diff
@@ -69,6 +69,7 @@ var (
 	)
 	availableWorkerCRINames = sets.NewString(
 		string(core.CRINameContainerD),
+		string(core.CRINameDocker),
```

Comment:
You can also extend `example/30-cloudprofile.yaml` and `example/90-shoot.yaml` and show that it is possible to set `docker` as cri name.

Reply:
I feel comfortable adding this to `example/30-cloudprofile.yaml` to list the possible configuration options and also hinting that people add `docker` to their `MachineImageVersion`s. For `example/90-shoot.yaml` I'm not quite sure if we'd want to do this:
* This would either mean adding a second `Worker` to the example or introduce some "either/or" configuration for the worker we currently have in there. Neither feels right from the point of view of keeping an example "simple".
* We probably don't want to actively encourage people to choose this configuration. 
WDYT?

Reply:
> We probably don't want to actively encourage people to choose this configuration.
WDYT?

Ah, ok, agree. Let's then don't mention about `docker` in `example/90-shoot.yaml`.